dev_guide/cicd.adoc

```diff
@@ -0,0 +1,254 @@
+[[dev-guide-cicd]]
+= CI / CD in OpenShift
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+{product-title} This next topic will focus on continuous integration and continuous delivery (`*CI/CD*`) of applications with OpenShift.  In particular, the goal is to
```

Comment:
product-title gets replaced w/ the product name (eg OpenShift Origin), not sure why you referenced it here.

s/this next topic/this topic/

s/OpenShift/{product-title}/